From f5f78dafab05fcfe36d0e5ea0fd18ce8b7368300 Mon Sep 17 00:00:00 2001
From: "Arun Kumar K.R" <akumarkr@codeaurora.org>
Date: Wed, 29 Jan 2014 13:08:48 -0800
Subject: [PATCH 4/5] display: remove compile time warnings

- remove unused variable in libexternal.
- make use of all variables in memtrack.

[haggertk: Picked to 8974. libexternal is N/A our family]

Change-Id: I67b34428f4a976327e4e9dcd6fd7dfb9063b969c
Signed-off-by: Kevin F. Haggerty <haggertk@lineageos.org>
---
 libmemtrack/memtrack_msm.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libmemtrack/memtrack_msm.c b/libmemtrack/memtrack_msm.c
index ac93f44f8..8adff965f 100644
--- a/libmemtrack/memtrack_msm.c
+++ b/libmemtrack/memtrack_msm.c
@@ -22,6 +22,8 @@
 
 int msm_memtrack_init(const struct memtrack_module *module)
 {
+    if(!module)
+        return -1;
     return 0;
 }
 
@@ -31,6 +33,8 @@ int msm_memtrack_get_memory(const struct memtrack_module *module,
                                 struct memtrack_record *records,
                                 size_t *num_records)
 {
+    if(!module)
+        return -1;
     if (type == MEMTRACK_TYPE_GL || type == MEMTRACK_TYPE_GRAPHICS) {
         return kgsl_memtrack_get_memory(pid, type, records, num_records);
     }
-- 
2.17.1

